physics/inertia_tensor.hpp

```diff
@@ -53,11 +56,19 @@ class InertiaTensor {
     Rotation<Frame, PrincipalAxesFrame> rotation;
   };
 
+  // A factory that creates an inertia tensor for a solid sphere of water having
+  // the given mass.  Useful e.g. for save compatibility.
+  static InertiaTensor<Frame> MakeSphericalInertiaTensor(Mass const& mass);
```

Comment:
Either pass ρ as a parameter, put "Water" in the name, or move this to ksp_plugin.